src/python/pants/net/http/fetcher.py

```diff
@@ -177,19 +177,19 @@ def recv_chunk(self, data):
         self.chunks = chunk_count
         if self.size:
           self._stream.write(b'\r')
-          self._stream.write('{:3}% '.format(int(self.read * 1.0 / self.size * 100)).encode())
+          self._stream.write(f"{int(self.read * 1.0 / self.size * 100):3}% ".encode())
         self._stream.write(b'.' * self.chunks)
         if self.size:
           size_width = len(str(self.download_size))
           downloaded = int(self.read / 1024)
-          self._stream.write('{} {} KB'.format(' ' * (self._width - self.chunks),
-                                               str(downloaded).rjust(size_width))
-                             .encode())
+          self._stream.write(
+            f"{' ' * (self._width - self.chunks)} {str(downloaded).rjust(size_width)} KB".encode()
+          )
         self._stream.flush()
 
     def finished(self):
       if self.chunks > 0:
-        self._stream.write(' {:.3f}s\n'.format(time.time() - self._start).encode())
+        self._stream.write(f" {time.time() - self._start:.3f}s\n".encode())
```

Comment:
Why the change to double quotes? We tend to prefer single quotes.

Reply:
Eh I'll change it back. Agreed for this case to go for smaller diff. 

Habit from Prettier (JS) and Black (Python) autoformatters both using double quotes by default because it's more likely you end up using `'` than having to escape `"` in strings.